util/java/libtiled-java/src/main/java/tiled/core/Map.java

```diff
@@ -44,38 +45,63 @@
  * @version 0.17
  */
 public class Map implements Iterable<MapLayer> {
+    // <editor-fold defaultstate="collapsed" desc="Enumerations">
+    public enum Orientation {
+        ORTHOGONAL,
+        ISOMETRIC,
+        /**
+        * Hexagonal.
+        *
+        * @since 0.11
+        */
+        HEXAGONAL,
+        /**
+        * Shifted/Staggered (used for iso and hex).
+        */
+        SHIFTED,
```

Comment:
`SHIFTED` was never used, and by the time I got around to implementing it, I called it `STAGGERED` instead, so you can remove this entry.